src/components/mediaViewer/mediaViewer.css

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.media-container {
+  cursor: pointer;
+}
+
+.video-container {
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+}
+
+.video-container:before {
+  content: "";
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: all .1s linear;
+  width: 100%;
+}
+
+.video-container:after {
+  background: #fff url(/img/video.svg) no-repeat 17px;
+  border-radius: 50%;
+  box-shadow: 0 2px 80px 0 hsl(0deg 0% 56% / 50%);
+  content: "";
+  height: 104px;
+  left: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%,-50%);
+  width: 104px;
+}
+
+.preview-container {
+  position: fixed;
+  z-index: 1000;
+}
+
+.preview-container > div {
+  background-color: rgba(0, 0, 0, 0.9) !important;;
+}
+
+.thumbnail {
+  max-height: 500px;
```

Comment:
Let's set the width to `100%` and height to `auto` as for md images rendered by default to fill in the full width between navigation.